openstack/blockstorage/noauth/requests.go

```diff
@@ -0,0 +1,57 @@
+package noauth
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+// EndpointOpts specifies a "noauth" Cinder Endpoint.
+type EndpointOpts struct {
+	// CinderEndpoint [required] is currently only used with "noauth" Cinder.
+	// A cinder endpoint with "auth_strategy=noauth" is necessary, for example:
+	// http://example.com:8776/v2.
+	CinderEndpoint string
+}
+
+/*
```

Comment:
This looks weird [to me] for such a small comment. Let's change to "//"-style like the others